source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -122,12 +188,26 @@ class ZooKeeperFilterConfig {
   const Stats::StatName unknown_scheme_rq_;
   const Stats::StatName unknown_opcode_latency_;
 
+  ErrorBudgetResponseType errorBudgetDecision(const OpCodes opcode,
+                                              const std::chrono::milliseconds& latency);
```

Comment:
we can just copy the latency. i.e., no '&'

Reply:
and make the funciton `const`

Reply:
Thanks, just made the change accordingly!